west.yml

```diff
@@ -54,7 +54,7 @@ manifest:
       revision: ca32746072ce3381f1c9ae46ba6cd34c69f8c0ee
       path: modules/crypto/mbedtls
     - name: mcumgr
-      revision: 84934959d2d1722a23b7e7e200191ae4a6f96168
+      revision: ee2cb08d76efa7791ac7a79fed5a73f549bf6abd
```

Comment:
Can you please set this to `pull/2/head`? Then CI can run on your PR and we know it is safe to merge the mcumgr PR.

After that is merged, you can set this SHA to the final commit which the maintainers have merged into the mcumgr repo in a force-push. But first let's see CI pass with it set to pull/2/head.

Thanks for all your patience! This is a new process for us.

Reply:
Do you means need twice commit for the west.yml, The first for CI, and then for actual merge?
I‘m sorry for I don't know how to set this to pull/2/head? Is there any case for reference?
If you can give guidance, I will be very grateful.

Reply:
Please see here: https://docs.zephyrproject.org/latest/guides/modules.html#changes-to-existing-module

You just need one commit, not two, so we can test the PR in the external module mcumgr. Then when the mcumgr PR is merged, you can force-push this commit to change pull/2/head to the final SHA which is merged into mcumgr. The commit ee2cb08d76efa7791ac7a79fed5a73f549bf6abd is not the commit which will actually get merged, since the final rebase committer will be different.

Reply:
@mbolivar  Sorry for late, I finally understand how to modify it. The CI look like build pass, but I haven't issue no, the gitlint fail.